simtools/applications/convert_geo_coordinates_of_array_elements.py

```diff
@@ -1,58 +1,53 @@
 #!/usr/bin/python3
-"""
-    Summary
-    -------
-    Convert and print a list of array element positions in different coordinate \
-    systems relevant for CTAO.
+"""Convert and print a list of array element positions in different coordinate systems.
 
     Description
     -----------
 
-    This application converts a list of array element positions in different CTAO \
-    coordinate systems.
-
+    Convert array element positions in different CTAO coordinate systems.
     Available coordinate systems are:
 
     1. UTM system
-    2. ground system (similar to sim_telarray system with x-axis pointing toward \
-        geographic north and y-axis pointing towards the west); altitude relative \
-        to the CORSIKA observation level.
+    2. ground system (similar to sim_telarray system with x-axis pointing toward geographic north
+       and y-axis pointing towards the west); altitude relative to the CORSIKA observation level.
+       Altitude is the height of the elevation rotation axis (plus some possible mirror offset).
```

Comment:
It is not clear to me what "plus some possible mirror offset" means. I think this refers to a possible shift due to the mirror shape, but it does not inform whether the shift is already included in the model, or how much is the shift? Should the user worry about this?

Reply:
My better suggestion is to remove `(plus some possible mirror offset)`. If this is an information the user needs, then I would pass the complete information, otherwise I think this piece of information is not needed here. This confuses more than adds to the docstring.